assets/src/application/ui/layout/entityList/RegistrationsLink/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { Groups } from '@appDisplay/icons';
+import { Tooltip } from '@infraUI/display';
+
+import './style.scss';
+
+interface Props {
+	href: string;
+	title: string;
+}
+
+const RegistrationsLink: React.FC<Props> = ({ href, title }) => {
+	return (
+		<Tooltip title={title}>
+			<a href={href} className={'ee-editor-details-reg-url-link'} target={'_blank'} rel={'noopener norefferer'}>
+				<Groups svgSize={24} />
+			</a>
+		</Tooltip>
+	);
+};
```

Comment:
nice...
I was just thinking to move that css into the `DateRegistrationsLink` component,
but I forgot that there was one for tickets too